pom.xml

```diff
@@ -233,6 +237,11 @@
         <artifactId>jnr-fuse</artifactId>
         <version>0.5.1</version>
       </dependency>
+      <dependency>
```

Comment:
lets try to move the catalog into its own top-level module, and move all the dependencies to the lowest level possible.

Reply:
can we avoid modifying this top level pom?